clang/lib/CodeGen/CGExprScalar.cpp

```diff
@@ -3446,6 +3446,8 @@ Value *ScalarExprEmitter::VisitOffsetOfExpr(OffsetOfExpr *E) {
       // Compute the index
       Expr *IdxExpr = E->getIndexExpr(ON.getArrayExprIndex());
       llvm::Value* Idx = CGF.EmitScalarExpr(IdxExpr);
+      if (Idx->getType()->isPointerTy())
+          Idx = CGF.getCapabilityIntegerValue(Idx);
```

Comment:
```suggestion
      if (Idx->getType()->isPointerTy())
        Idx = CGF.getCapabilityIntegerValue(Idx);
```
this will fix the clang-format warning.